src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -888,21 +909,23 @@ class RsMetaItemStub(
             with(dataStream) {
                 writeName(stub.referenceName)
                 writeBoolean(stub.hasEq)
-                writeUTFFast(stub.value ?: "")
+                writeUTFFast(stub.value)
             }
 
         override fun deserialize(dataStream: StubInputStream, parentStub: StubElement<*>?): RsMetaItemStub =
             RsMetaItemStub(parentStub, this,
                 dataStream.readNameAsString()!!,
                 dataStream.readBoolean(),
-                dataStream.readUTFFast().let { if (it == "") null else it })
+                dataStream.readUTFFastAsNullable())
 
         override fun indexStub(stub: RsMetaItemStub, sink: IndexSink) {
         }
     }
 }
 
 private fun StubInputStream.readNameAsString(): String? = readName()?.string
+private fun StubInputStream.readUTFFastAsNullable(): String? = readUTFFast().let { if (it == "") null else it }
 
 private fun <E : Enum<E>> StubOutputStream.writeEnum(e: E) = writeByte(e.ordinal)
 private fun <E : Enum<E>> StubInputStream.readEnum(values: Array<E>) = values[readByte().toInt()]
+private fun StubOutputStream.writeUTFFast(value: String?) = writeUTFFast(value ?: "")
```

Comment:
Nit: let's name it `writeUTFFastAsNullable`, to make sure that read and write sides are symmetric.